### PR TITLE
include HTML charset

### PR DESCRIPTION
**Pull request by asgs**
## Proposed Changes
The static HTML pages don't include a directive indicating the charset of the page content. This triggers an error statement printed in the developer console (though it's perfectly normal for the content at hand). So include a `<meta>` directive specifying UTF-8 charset.

## Types of Changes

- [ ] Bugfix (non-breaking change which fixes issue #NNNN)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (correction or otherwise)
- [x] Cosmetics (whitespace, appearance)

## Checklist

- [x] I have read the `CONTRIBUTING.md` document
- [x] I have signed the CA (see https://cla.pivotal.io/sign/rabbitmq)
- [ ] All tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in related repositories

## Further Comments

Nothing else to ponder about.

### CODE REVIEW

File: priv/www/index.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!doctype html>
 <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (lukebakken) [inline priv/www/index.html]: I wonder if this tag should be in the other file, or removed completely - https://stackoverflow.com/a/6771584

Comment (asgs) [reply priv/www/index.html]: this tag was already present and going through the SO post you linked, it seems it's only necessary for IE9 and versions below. What's your take?

**Comment by michaelklishin** [comment]
There are folks who would appreciate if we didn’t drop IE9 support at this point. Older versions can be safely ignored.

File: priv/www/index.html (hunk 1 above)
Comment (michaelklishin) [reply priv/www/index.html]: There are folks who would appreciate if we didn’t drop IE9 support at this point. Older versions can be safely ignored.

**Comment by lukebakken** [comment]
@michaelklishin - tested on Win 8.1 with IE 11, and Chromium 64 on linux

**Pull request merged**